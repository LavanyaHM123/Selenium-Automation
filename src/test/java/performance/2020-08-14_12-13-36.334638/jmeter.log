2020-08-14 12:14:33,297 INFO o.a.j.u.JMeterUtils: Setting Locale to en_EN
2020-08-14 12:14:33,390 INFO o.a.j.JMeter: Loading user properties from: C:\Users\Prakat-L-0005\.bzt\jmeter-taurus\5.2.1\bin\user.properties
2020-08-14 12:14:33,392 INFO o.a.j.JMeter: Loading system properties from: C:\Users\Prakat-L-0005\.bzt\jmeter-taurus\5.2.1\bin\system.properties
2020-08-14 12:14:33,393 INFO o.a.j.JMeter: Loading additional properties from: C:\Users\Prakat-L-0005\Desktop\Automation pipeline\e2e-automation-pipeline-master\src\test\java\performance\2020-08-14_12-13-36.334638\jmeter-bzt.properties
2020-08-14 12:14:33,400 INFO o.a.j.JMeter: Setting System properties from file: C:\Users\Prakat-L-0005\Desktop\Automation pipeline\e2e-automation-pipeline-master\src\test\java\performance\2020-08-14_12-13-36.334638\system.properties
2020-08-14 12:14:33,406 INFO o.a.j.JMeter: Copyright (c) 1998-2019 The Apache Software Foundation
2020-08-14 12:14:33,406 INFO o.a.j.JMeter: Version 5.2.1
2020-08-14 12:14:33,407 INFO o.a.j.JMeter: java.version=1.8.0_202
2020-08-14 12:14:33,407 INFO o.a.j.JMeter: java.vm.name=Java HotSpot(TM) 64-Bit Server VM
2020-08-14 12:14:33,407 INFO o.a.j.JMeter: os.name=Windows 10
2020-08-14 12:14:33,407 INFO o.a.j.JMeter: os.arch=amd64
2020-08-14 12:14:33,407 INFO o.a.j.JMeter: os.version=10.0
2020-08-14 12:14:33,407 INFO o.a.j.JMeter: file.encoding=Cp1252
2020-08-14 12:14:33,408 INFO o.a.j.JMeter: java.awt.headless=true
2020-08-14 12:14:33,408 INFO o.a.j.JMeter: Max memory     =1073741824
2020-08-14 12:14:33,410 INFO o.a.j.JMeter: Available Processors =4
2020-08-14 12:14:33,428 INFO o.a.j.JMeter: Default Locale=English (EN)
2020-08-14 12:14:33,430 INFO o.a.j.JMeter: JMeter  Locale=English (EN)
2020-08-14 12:14:33,430 INFO o.a.j.JMeter: JMeterHome=C:\Users\Prakat-L-0005\.bzt\jmeter-taurus\5.2.1
2020-08-14 12:14:33,431 INFO o.a.j.JMeter: user.dir  =C:\Users\Prakat-L-0005\Desktop\Automation pipeline\e2e-automation-pipeline-master\src\test\java\performance
2020-08-14 12:14:33,437 INFO o.a.j.JMeter: PWD       =C:\Users\Prakat-L-0005\Desktop\Automation pipeline\e2e-automation-pipeline-master\src\test\java\performance
2020-08-14 12:14:37,950 INFO o.a.j.JMeter: IP: 192.168.99.1 Name: Prakat-L005 FullName: Prakat-L005
2020-08-14 12:14:37,950 INFO o.a.j.JMeter: user.classpath=C:/Users/Prakat-L-0005/Desktop/Automation pipeline/e2e-automation-pipeline-master/src/test/java/performance/2020-08-14_12-13-36.334638;C:/Users/Prakat-L-0005/Desktop/Automation pipeline/e2e-automation-pipeline-master/src/test/java/performance/2020-08-14_12-13-36.334638
2020-08-14 12:14:37,951 INFO o.a.j.JMeter: Adding to classpath and loader: C:/Users/Prakat-L-0005/Desktop/Automation pipeline/e2e-automation-pipeline-master/src/test/java/performance/2020-08-14_12-13-36.334638
2020-08-14 12:14:37,956 INFO o.a.j.JMeter: Adding to classpath and loader: C:/Users/Prakat-L-0005/Desktop/Automation pipeline/e2e-automation-pipeline-master/src/test/java/performance/2020-08-14_12-13-36.334638
2020-08-14 12:14:37,982 INFO o.a.j.s.FileServer: Default base='C:\Users\Prakat-L-0005\Desktop\Automation pipeline\e2e-automation-pipeline-master\src\test\java\performance'
2020-08-14 12:14:38,132 INFO o.a.j.s.FileServer: Set new base='C:\Users\Prakat-L-0005\Desktop\Automation pipeline\e2e-automation-pipeline-master\src\test\java\performance\2020-08-14_12-13-36.334638'
2020-08-14 12:14:38,429 INFO o.a.j.s.SaveService: Testplan (JMX) version: 2.2. Testlog (JTL) version: 2.2
2020-08-14 12:14:38,445 INFO o.a.j.s.SaveService: Using SaveService properties file encoding UTF-8
2020-08-14 12:14:38,468 INFO o.a.j.s.SaveService: Using SaveService properties version 5.0
2020-08-14 12:14:38,485 INFO o.a.j.s.SaveService: Loading file: C:\Users\Prakat-L-0005\Desktop\Automation pipeline\e2e-automation-pipeline-master\src\test\java\performance\2020-08-14_12-13-36.334638\modified_requests.jmx
2020-08-14 12:14:38,737 INFO o.a.j.p.h.c.CacheManager: Will only cache the following methods: [GET]
2020-08-14 12:14:38,741 INFO o.a.j.p.h.c.CookieManager: Settings: Delete null: true Check: true Allow variable: true Save: false Prefix: COOKIE_
2020-08-14 12:14:38,781 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for text/html is org.apache.jmeter.protocol.http.parser.LagartoBasedHtmlParser
2020-08-14 12:14:38,782 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for application/xhtml+xml is org.apache.jmeter.protocol.http.parser.LagartoBasedHtmlParser
2020-08-14 12:14:38,782 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for application/xml is org.apache.jmeter.protocol.http.parser.LagartoBasedHtmlParser
2020-08-14 12:14:38,782 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for text/xml is org.apache.jmeter.protocol.http.parser.LagartoBasedHtmlParser
2020-08-14 12:14:38,782 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for text/vnd.wap.wml is org.apache.jmeter.protocol.http.parser.RegexpHTMLParser
2020-08-14 12:14:38,783 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for text/css is org.apache.jmeter.protocol.http.parser.CssParser
2020-08-14 12:14:39,127 INFO o.a.j.e.StandardJMeterEngine: Running the test!
2020-08-14 12:14:39,161 INFO o.a.j.s.SampleEvent: List of sample_variables: []
2020-08-14 12:14:39,161 INFO o.a.j.s.SampleEvent: List of sample_variables: []
2020-08-14 12:14:39,180 INFO o.a.j.e.u.CompoundVariable: Note: Function class names must contain the string: '.functions.'
2020-08-14 12:14:39,180 INFO o.a.j.e.u.CompoundVariable: Note: Function class names must not contain the string: '.gui.'
2020-08-14 12:14:42,254 INFO o.a.j.JMeter: Running test (1597387482254)
2020-08-14 12:14:42,401 INFO o.a.j.e.StandardJMeterEngine: Starting ThreadGroup: 1 : basic-performance-test
2020-08-14 12:14:42,401 INFO o.a.j.e.StandardJMeterEngine: Starting 10 threads for group basic-performance-test.
2020-08-14 12:14:42,401 INFO o.a.j.e.StandardJMeterEngine: Thread will continue on error
2020-08-14 12:14:42,440 INFO o.a.j.e.StandardJMeterEngine: All thread groups have been started
2020-08-14 12:14:42,440 INFO c.b.j.t.c.ConcurrencyThreadStarter: Start supplying threads
2020-08-14 12:14:45,468 INFO o.a.j.s.SampleResult: Note: Sample TimeStamps are START times
2020-08-14 12:14:45,468 INFO o.a.j.s.SampleResult: sampleresult.default.encoding is set to UTF-8
2020-08-14 12:14:45,469 INFO o.a.j.s.SampleResult: sampleresult.useNanoTime=true
2020-08-14 12:14:45,469 INFO o.a.j.s.SampleResult: sampleresult.nanoThreadSleep=5000
2020-08-14 12:14:45,476 INFO o.a.j.t.JMeterThread: Thread started: basic-performance-test-ThreadStarter 1-1
2020-08-14 12:14:46,336 INFO o.a.j.p.h.s.HTTPHCAbstractImpl: Local host = Prakat-L005
2020-08-14 12:14:46,348 INFO o.a.j.p.h.s.HTTPHC4Impl: HTTP request retry count = 0
2020-08-14 12:14:47,580 INFO o.a.j.p.h.s.h.LazyLayeredConnectionSocketFactory: Setting up HTTPS TrustAll Socket Factory
2020-08-14 12:14:47,596 INFO o.a.j.u.JsseSSLManager: Using default SSL protocol: TLS
2020-08-14 12:14:47,597 INFO o.a.j.u.JsseSSLManager: SSL session context: per-thread
2020-08-14 12:14:49,773 INFO o.a.j.u.SSLManager: JmeterKeyStore Location:  type JKS
2020-08-14 12:14:49,777 INFO o.a.j.u.SSLManager: KeyStore created OK
2020-08-14 12:14:49,777 WARN o.a.j.u.SSLManager: Keystore file not found, loading empty keystore
2020-08-14 12:14:50,883 INFO o.a.j.p.h.p.BaseParser: Created org.apache.jmeter.protocol.http.parser.LagartoBasedHtmlParser
2020-08-14 12:14:50,977 INFO o.a.j.p.h.s.ResourcesDownloader: Creating ResourcesDownloader with keepalive_inseconds : 60
2020-08-14 12:14:51,482 INFO o.a.j.t.JMeterThread: Thread started: basic-performance-test-ThreadStarter 1-2
2020-08-14 12:14:52,977 INFO o.a.j.p.h.p.BaseParser: Created org.apache.jmeter.protocol.http.parser.CssParser
2020-08-14 12:14:57,543 INFO o.a.j.t.JMeterThread: Thread started: basic-performance-test-ThreadStarter 1-3
2020-08-14 12:15:03,503 INFO o.a.j.t.JMeterThread: Thread started: basic-performance-test-ThreadStarter 1-4
2020-08-14 12:15:09,508 INFO o.a.j.t.JMeterThread: Thread started: basic-performance-test-ThreadStarter 1-5
2020-08-14 12:15:15,515 INFO o.a.j.t.JMeterThread: Thread started: basic-performance-test-ThreadStarter 1-6
2020-08-14 12:15:21,519 INFO o.a.j.t.JMeterThread: Thread started: basic-performance-test-ThreadStarter 1-7
2020-08-14 12:15:27,524 INFO o.a.j.t.JMeterThread: Thread started: basic-performance-test-ThreadStarter 1-8
2020-08-14 12:15:33,528 INFO o.a.j.t.JMeterThread: Thread started: basic-performance-test-ThreadStarter 1-9
2020-08-14 12:15:39,539 INFO o.a.j.t.JMeterThread: Thread started: basic-performance-test-ThreadStarter 1-10
2020-08-14 12:17:12,603 INFO c.b.j.t.c.ConcurrencyThreadStarter: Done supplying threads
2020-08-14 12:17:12,658 INFO o.a.j.t.JMeterThread: Thread is done: basic-performance-test-ThreadStarter 1-9
2020-08-14 12:17:12,658 INFO o.a.j.t.JMeterThread: Thread finished: basic-performance-test-ThreadStarter 1-9
2020-08-14 12:17:16,085 INFO o.a.j.t.JMeterThread: Thread is done: basic-performance-test-ThreadStarter 1-1
2020-08-14 12:17:16,085 INFO o.a.j.t.JMeterThread: Thread finished: basic-performance-test-ThreadStarter 1-1
2020-08-14 12:17:18,242 INFO o.a.j.t.JMeterThread: Thread is done: basic-performance-test-ThreadStarter 1-4
2020-08-14 12:17:18,242 INFO o.a.j.t.JMeterThread: Thread finished: basic-performance-test-ThreadStarter 1-4
2020-08-14 12:17:18,998 INFO o.a.j.t.JMeterThread: Thread is done: basic-performance-test-ThreadStarter 1-8
2020-08-14 12:17:18,998 INFO o.a.j.t.JMeterThread: Thread finished: basic-performance-test-ThreadStarter 1-8
2020-08-14 12:17:22,430 INFO o.a.j.t.JMeterThread: Thread is done: basic-performance-test-ThreadStarter 1-5
2020-08-14 12:17:22,430 INFO o.a.j.t.JMeterThread: Thread finished: basic-performance-test-ThreadStarter 1-5
2020-08-14 12:17:23,729 INFO o.a.j.t.JMeterThread: Thread is done: basic-performance-test-ThreadStarter 1-3
2020-08-14 12:17:23,729 INFO o.a.j.t.JMeterThread: Thread finished: basic-performance-test-ThreadStarter 1-3
2020-08-14 12:17:26,257 INFO o.a.j.t.JMeterThread: Thread is done: basic-performance-test-ThreadStarter 1-10
2020-08-14 12:17:26,258 INFO o.a.j.t.JMeterThread: Thread finished: basic-performance-test-ThreadStarter 1-10
2020-08-14 12:17:33,864 INFO o.a.j.t.JMeterThread: Thread is done: basic-performance-test-ThreadStarter 1-6
2020-08-14 12:17:33,865 INFO o.a.j.t.JMeterThread: Thread finished: basic-performance-test-ThreadStarter 1-6
2020-08-14 12:17:37,136 INFO o.a.j.t.JMeterThread: Thread is done: basic-performance-test-ThreadStarter 1-2
2020-08-14 12:17:37,137 INFO o.a.j.t.JMeterThread: Thread finished: basic-performance-test-ThreadStarter 1-2
2020-08-14 12:17:44,138 INFO o.a.j.t.JMeterThread: Thread is done: basic-performance-test-ThreadStarter 1-7
2020-08-14 12:17:44,138 INFO o.a.j.t.JMeterThread: Thread finished: basic-performance-test-ThreadStarter 1-7
2020-08-14 12:17:44,141 INFO o.a.j.e.StandardJMeterEngine: Notifying test listeners of end of test
2020-08-14 12:17:44,149 INFO o.a.j.e.StandardJMeterEngine: Forced JVM shutdown requested at end of test
