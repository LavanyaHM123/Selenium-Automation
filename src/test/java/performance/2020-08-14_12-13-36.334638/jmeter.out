Created the tree successfully using C:\Users\Prakat-L-0005\Desktop\Automation pipeline\e2e-automation-pipeline-master\src\test\java\performance\2020-08-14_12-13-36.334638\modified_requests.jmx
Starting standalone test @ Fri Aug 14 12:14:39 IST 2020 (1597387479123)
Waiting for possible Shutdown/StopTestNow/HeapDump/ThreadDump message on port 4445
Tidying up ...    @ Fri Aug 14 12:17:44 IST 2020 (1597387664148)
... end of run
